### PR TITLE
Adds article on running OpenFaaS on Lambda

### PR DESCRIPTION
**Pull request by ewilde**
Signed-off-by: Edward Wilde <ewilde@gmail.com>

### CODE REVIEW

**Comment by alexellis** [comment]
@ewilde is this a draft, or would you like me to review?

File: _posts/2019-05-24-announcing-lambda.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: Announcing support to run OpenFaaS functions on AWS Lambda
+description: Edward Wilde introduces faas-lambda a way to run OpenFaaS functions on AWS Lambda. Learn how simple container based functions can be deployed to AWS Lambda and Kubernetes with one seamless developer experience
+date: 2019-05-24
+image: /images/openfaas-on-fargate/astronomy-atmosphere-earth-220201.jpg
+categories:
+  - fargate
```
Comment (alexellis) [inline _posts/2019-05-24-announcing-lambda.md]: `fargate` -> `lambda`

File: _posts/2019-05-24-announcing-lambda.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+---
+title: Announcing support to run OpenFaaS functions on AWS Lambda
+description: Edward Wilde introduces faas-lambda a way to run OpenFaaS functions on AWS Lambda. Learn how simple container based functions can be deployed to AWS Lambda and Kubernetes with one seamless developer experience
+date: 2019-05-24
+image: /images/openfaas-on-fargate/astronomy-atmosphere-earth-220201.jpg
```
Comment (alexellis) [inline _posts/2019-05-24-announcing-lambda.md]: Perhaps a new photo?

File: _posts/2019-05-24-announcing-lambda.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+author_staff_member: ed
+dark_background: true
+---
+
+In this post we are going to examine some of the possible use cases for running OpenFaaS functions on AWS Lambda.
+Then we are going to take a look at some of the new components I've written to make this possible.
+Finally I would like to ask the community for some feedback on these new ideas.
+
+
+## Why would you use OpenFaaS with AWS Lambda?
```
Comment (alexellis) [inline _posts/2019-05-24-announcing-lambda.md]: `## Why would you use OpenFaaS with AWS Lambda?`

`## What value is there in combining OpenFaaS with AWS Lambda?`

File: _posts/2019-05-24-announcing-lambda.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+There could be many great reasons to run OpenFaaS functions on Lambda here are some that I've
+been considering. 
+
+**CI/CD and a unified developer experience**
```
Comment (alexellis) [inline _posts/2019-05-24-announcing-lambda.md]: Let's try `*` bullets

File: _posts/2019-05-24-announcing-lambda.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+until you are ready to migrate them over to Kubernetes
+
+**Proprietary integrations**
+There are some integration points in the AWS architecture that mandate the use of Lambdas. 
+However, there is no reason you can't create these function using OpenFaaS and it's enhanced
+developer experience.
+
+
+## How does it work?
+Hopefully I've peaked your interest? If we examine the diagram below, you can see that the
```
Comment (alexellis) [inline _posts/2019-05-24-announcing-lambda.md]: `Hopefully I've peaked your interest?` <- let's remove.

**Comment by ewilde** [comment]
@alexellis please review, be useful to get early feedback

File: _posts/2019-05-24-announcing-lambda.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+faas-lambda provider is a standard OpenFaaS component that implements the [gateway / provider contract](link).
+The provider understands how to translate requests from the gateway into appropriate 
+responses in the AWS Lambda world. In this architecture the faas-lambda provider runs in Kubernetes
+and using the go-lang AWS SDK communicates with the AWS Lambda service APIs. Credentials to enable that communication
+are stored securely in Kubernetes as secrets, ?????or inherited transparent using IAM roles if running
```
Comment (alexellis) [inline _posts/2019-05-24-announcing-lambda.md]: `?????` ?

File: _posts/2019-05-24-announcing-lambda.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+on EKS???????.
+
+![the faas-lambda provider architecture](/images/faas-lambda/faas-lambda-provider.png)
+
+### Deploying a function
```
Comment (alexellis) [inline _posts/2019-05-24-announcing-lambda.md]: As per style guide (see README.md), please use imperative -> `Deploy a function` / `Invoke a function`

File: _posts/2019-05-24-announcing-lambda.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+So that we can run both OpenFaaS functions on AWS Lambda our normal execution platform Kubernetes
+we needed to introduce a new component into the architecture. This component is called the
+`faas-federator`.
+
+![enable multiple execution platform with faas-federator]() 
+
+## What's next
+This project is in early release. As such we would like to get feedback from the community:
+
+- Please email [] if you would like to try the new provider
```
Comment (alexellis) [inline _posts/2019-05-24-announcing-lambda.md]: Please use a Google form instead.

Comment (alexellis) [reply _posts/2019-05-24-announcing-lambda.md]: https://forms.gle/Ug5XCZjgFxXfm3HL8

File: _posts/2019-05-24-announcing-lambda.md (hunk 1 above)
Comment (ewilde) [reply _posts/2019-05-24-announcing-lambda.md]: change made

File: _posts/2019-05-24-announcing-lambda.md (hunk 2 above)
Comment (ewilde) [reply _posts/2019-05-24-announcing-lambda.md]: change made

File: _posts/2019-05-24-announcing-lambda.md (hunk 3 above)
Comment (ewilde) [reply _posts/2019-05-24-announcing-lambda.md]: change made

File: _posts/2019-05-24-announcing-lambda.md (hunk 4 above)
Comment (ewilde) [reply _posts/2019-05-24-announcing-lambda.md]: change made

File: _posts/2019-05-24-announcing-lambda.md (hunk 5 above)
Comment (ewilde) [reply _posts/2019-05-24-announcing-lambda.md]: change made

File: _posts/2019-05-24-announcing-lambda.md (hunk 6 above)
Comment (ewilde) [reply _posts/2019-05-24-announcing-lambda.md]: removed placeholder ???? linked to iam solution for EKS

File: _posts/2019-05-24-announcing-lambda.md (hunk 7 above)
Comment (ewilde) [reply _posts/2019-05-24-announcing-lambda.md]: change made

File: _posts/2019-05-24-announcing-lambda.md (hunk 8 above)
Comment (ewilde) [reply _posts/2019-05-24-announcing-lambda.md]: link added to form 👏

**Comment by ewilde** [comment]
@alexellis all conversation resolved, please take another look

**Pull request closed without merging**